### PR TITLE
Extra attempt reason recording

### PR DESCRIPTION
**Pull request by hanwu85**
Recommending the delegates to record the reasons for extra attempts.

### CODE REVIEW

**Comment by maconard** [comment]
We always record the reasons for extras on the back of the affected scorecard at Midwest comps :)

**Comment by xsrvmy** [comment]
> We always record the reasons for extras on the back of the affected scorecard at Midwest comps :)

I have the same experience here in Ontario.

File: wca-guidelines.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ To be more informative, each Guideline is classified using one of the following
 ## <article-11><incidents><incidents> Article 11: Incidents
 
 - 11e+) [CLARIFICATION] Since an appeal is not guaranteed to be successful, the competitor may choose to keep the timer running while appealing it, and resume the attempt when appropriate.
+- 11e++) [RECOMMENDATION] The deleagte who grants the extra attempt should record the reason(s) for the extra attempt (e.g. on the back of the score sheet).
```
Comment (Ivan-Brigidano) [inline wca-guidelines.md]:
```suggestion
- 11e++) [RECOMMENDATION] The WCA Delegate should record the reason(s) for the extra attempt (e.g. on the back of the score sheet).
```

**Review by Ivan-Brigidano — CHANGES_REQUESTED** [review]

**Comment by Ivan-Brigidano** [comment]
@maconard that's a good habit!
@hanwu85 important typo: ~delegate~ --> WCA Delegate. We must use an [uniform style](https://github.com/thewca/wca-regulations/blob/draft/style-guide.md).

**Comment by hanwu85** [comment]
Thanks!  I just roughly checked the original document.
E2c2++ also needs to be updated to **the WCA Delegat**

**Review by Ivan-Brigidano — APPROVED** [review]

**Review by lgarron — APPROVED** [review]

**Pull request merged**